README.md

```diff
@@ -60,6 +60,46 @@ If you're on Linux, you'll probably be able to work it out.
 
 Windows users approach with caution – we've been able to get pa11y running (Windows 7, Node 0.12) but only after installing Visual Studio and the Windows SDK (as well as Git, Python and PhantomJS). The [Windows installation instructions for node-gyp][windows-install] are a good place to start.
 
+If you are able to successfully install both pa11y and PhantomJS on Windows but run into following error
+
+``` bash
+> pa11y google.com
```

Comment:
Could we change the two `google.com` references in your examples to `nature.com` for consistency with the rest of the docs?